modules/cli/src/main/scala/scala/cli/commands/default/Default.scala

```diff
@@ -0,0 +1,104 @@
+package scala.cli.commands.default
+
+import caseapp.core.RemainingArgs
+
+import java.io.File
+
+import scala.cli.commands.ScalaCommand
+import scala.cli.internal.Constants
+import scala.util.Using
+
+object Default extends ScalaCommand[DefaultOptions] {
+
+  private def readDefaultFile(path: String): Array[Byte] = {
+    val resourcePath = Constants.defaultFilesResourcePath + "/" + path
+    val cl           = Thread.currentThread().getContextClassLoader
+    val resUrl       = cl.getResource(resourcePath)
+    if (resUrl == null)
+      sys.error(s"Should not happen - resource $resourcePath not found")
+    Using.resource(resUrl.openStream())(_.readAllBytes())
+  }
+
+  final case class DefaultFile(
+    path: os.SubPath,
+    content: () => Array[Byte]
+  ) {
+    def printablePath = path.segments.mkString(File.separator)
+  }
+
+  def defaultWorkflow: Array[Byte] =
+    readDefaultFile("workflows/default.yml")
+  def defaultGitignore: Array[Byte] =
+    readDefaultFile("gitignore")
+
+  val defaultFiles = Map(
+    "workflow"  -> DefaultFile(os.sub / ".github" / "workflows" / "ci.yml", () => defaultWorkflow),
+    "gitignore" -> DefaultFile(os.sub / ".gitignore", () => defaultGitignore)
+  )
+  val defaultFilesByRelPath = defaultFiles.flatMap {
+    case (_, d) =>
+      // d.path.toString and d.printablePath differ on Windows (one uses '/', the other '\')
+      Seq(
+        d.path.toString -> d,
+        d.printablePath -> d
+      )
+  }
+
+  private def unrecognizedFile(name: String): Nothing = {
+    System.err.println(
```

Comment:
hmm... should we prefer using a logger rather than just stderr/stdout?
(it's been discussed, but I don't think we've agreed on a convention, really, just re-raising it here)

Reply:
I was unsure about using a logger here, as this wasn't meant to print much… I changed that, but I added a new `Logger.error` method, for some messages here (that should be printed even when `-q` is passed).